poetry.lock

```diff
@@ -735,17 +735,16 @@ validation = ["swagger-spec-validator (>=2.1.0)"]
 
 [[package]]
 name = "flake8"
-version = "3.9.2"
-description = "the modular source code checker: pep8 pyflakes and co"
+version = "2.3.0"
```

Comment:
This is an odd dependency to downgrade - maybe need to regenerate the lock file, as I can't see why updating the thycotic/delinea SDK should cause this to happen?

Reply:
@glennmatthews 

I've done the following:

- Open a new shell and execute `deactivate` to ensure no virtual environment is active
- `git checkout upstream/develop pyproject.toml`
- `git checkout upstream/develop poetry.lock`
- `poetry lock`

The same downgrade is there. I have updated pyproject.toml to restrict downgrade

```
[tool.poetry.dev-dependencies]
flake8 = ">=3.9.2"
```

Reply:
Good enough! Weird that it was needed though.